crates/typst-library/src/lib.rs

```diff
@@ -63,8 +63,12 @@ fn prelude(global: &mut Scope) {
     global.define("green", Color::GREEN);
     global.define("lime", Color::LIME);
     global.define("luma", Color::luma_data());
+    global.define("oklab", Color::oklab_data());
     global.define("rgb", Color::rgb_data());
+    global.define("linear-rgb", Color::linear_rgb_data());
     global.define("cmyk", Color::cmyk_data());
+    global.define("hsl", Color::hsl_data());
+    global.define("hsv", Color::hsv_data());
```

Comment:
In this case, we should remove `oklab` as well. If the color space turns out to be popular for color definitions, we can still re-export it globally. It may be worth to re-export either HSL or HSV. I personally think HSL is the better space, but HSV is more widely used. What do you think?